app/views/projects/samples/_table.html.erb

```diff
@@ -33,39 +36,42 @@
     <% samples.each do |sample| %>
       <tr
         id="<%= sample.id %>"
-        class="text-sm font-normal text-slate-500 dark:text-slate-400"
+        class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
       >
-        <% if allowed_to?(:transfer_sample?, project) %>
-          <td class="w-10 ps-4">
+        <td class="px-6 py-3 whitespace-nowrap sticky left-0 bg-slate-50 dark:bg-slate-900">
+          <% if allowed_to?(:transfer_sample?, project) %>
             <%= check_box_tag "sample_ids[]",
-            sample.id,
-            nil,
-            id: dom_id(sample),
-            "aria-label": sample.name,
-            data: {
-              action: "input->selection#toggle",
-              selection_target: "rowSelection"
-            },
-            class:
-              "w-4 h-4 text-primary-600 bg-slate-100 border-slate-300 rounded focus:ring-primary-500 dark:focus:ring-primary-600 dark:ring-offset-slate-800 focus:ring-2 dark:bg-slate-700 dark:border-slate-600" %>
-          </td>
-        <% end %>
-        <td class="px-4 py-3 whitespace-nowrap">
+                              sample.id,
+                              nil,
+                              id: dom_id(sample),
+                              "aria-label": sample.name,
+                              data: {
+                                action: "input->selection#toggle",
+                                selection_target: "rowSelection"
+                              },
+                              class:
+                                "w-4 h-4 mr-2.5 text-primary-600 bg-slate-100 border-slate-300 rounded focus:ring-primary-500 dark:focus:ring-primary-600 dark:ring-offset-slate-800 focus:ring-2 dark:bg-slate-700 dark:border-slate-600" %>
+          <% end %>
           <%= link_to sample.name,
           namespace_project_sample_path(id: sample.id),
           data: {
             turbo: false
           },
-          class: "text-grey-900 dark:text-grey-100 font-semibold hover:underline" %>
+                      class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
         </td>
-        <td class="px-4 py-3">
+        <td class="px-6 py-3 whitespace-nowrap">
           <%= l(sample.created_at.localtime, format: :full_date) %>
         </td>
-        <td class="px-4 py-3">
+        <td class="px-6 py-3 whitespace-nowrap">
           <%= viral_time_ago(original_time: sample.updated_at) %>
         </td>
+        <% @fields.each do |column| %>
+          <td class="px-6 py-3 whitespace-nowrap">
+            <%= sample[column].present? ? sample[column] : sample.metadata[column] %>
```

Comment:
**nitpick (blocking):** What is this doing? The only values in `@fields` will be keys present in the metadata so this check is redundant.

Reply:
Updated in 2b505ec6665d25ab9f6f8725b9350582067b7efa